akka-actor-typed/src/main/scala/akka/actor/typed/delivery/internal/ProducerControllerImpl.scala

```diff
@@ -537,7 +536,7 @@ private class ProducerControllerImpl[A: ClassTag](
         consumerController,
         newFirstSeqNr)
       if (s.unconfirmed.nonEmpty) {
-        timers.startTimerWithFixedDelay(ResendFirst, ResendFirst, 1.second)
+        timers.startTimerWithFixedDelay(ResendFirst, delay = settings.durableQueueResendFirstInterval)
         context.self ! ResendFirst
```

Comment:
By the way, why `ResendFirst` is sent to self since it is scheduled one line above?

Reply:
to resend immediately, otherwise it would be an initial delay of 1 sec